gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/bundle/entity/ClusterProperty.java

```diff
@@ -12,6 +12,8 @@
 
 @Named("CLUSTER_PROPERTY")
 public class ClusterProperty implements Entity {
+    public static final String CLUSTER_HOSTNAME_PROPERTY_NAME = "cluster.hostname";
```

Comment:
there is a PropertyConstants class, maybe this can go there

Reply:
This isn't a property, it's the name of a cluster property entity. Similar to `com.ca.apim.gateway.cagatewayexport.tasks.explode.bundle.entity.PrivateKeyEntity#SSL_DEFAULT_PRIVATE_KEY` or `com.ca.apim.gateway.cagatewayexport.tasks.explode.bundle.entity.IdentityProviderEntity#INTERNAL_IDP_NAME`